tests/conftest.py

```diff
@@ -60,3 +62,27 @@ def __getattribute__(self, name):
                 return getattr(monkeypatch, name)
 
     return Monkeypatcher()
+
+
+@pytest.fixture
+def config(tmp_path):
+    """Provide a config class with an extra set method for the test to change it."""
+
+    class TestConfig(config_module._Config):
+        """The Config, but with a method to set test values."""
+
+        def set(self, **kwargs):
```

Comment:
IMO, now that Config is a proper object, this fixture should no longer be used.  The tests should instantiate their own Config() with the necessary parameters required for that test rather than jump through this hoop.

Reply:
As the config is not omnipresent anymore, the config needs to be used in ~130 tests; having it as a fixture, easy to use in most of the cases where a special value is not needed reduces a lot the code needed in all the tests.